app/models/report_composer.rb

```diff
@@ -209,6 +228,8 @@ def schedule_rendering
   end
 
   def render(mode: Foreman::Renderer::REAL_MODE, **params)
+    params[:variables] ||= {}
```

Comment:
Wouldn't it make more sense (and probably look cleaner) to use `params:` instead of `variables`?
Than we get rid of the `__` and we can just get it as `params[:format]`

Reply:
you're right, moved to params